Library/Homebrew/cmd/update-report.rb

```diff
@@ -144,6 +144,18 @@ def update_report
     link_completions_manpages_and_docs
     Tap.each(&:link_completions_and_manpages)
 
+    failed_fetch_dirs = ENV["HOMEBREW_FAILED_FETCH_DIRS"]&.split("\n")
+    if failed_fetch_dirs.present?
+      failed_fetch_taps = failed_fetch_dirs.map { |dir| Tap.from_path(dir) }
+
+      puts Formatter.headline "Some taps failed to update!", color: :red
+      puts <<~EOS
+        The following taps failed to update:
+          #{failed_fetch_taps.join("\n  ")}
+        Try running `brew tap --repair`
```

Comment:
I'm always unsure about the use of "try". When do we expect this to work/not? Could we capture the actual error message (perhaps using `tee`) to identify the specific "the branch isn't there" case?